core/src/test/java/org/apache/calcite/test/ExceptionMessageTest.java

```diff
@@ -141,4 +167,29 @@ private void runQuery(String sql) throws SQLException {
           containsString("Object 'nonexistentTable' not found"));
     }
   }
+
+  @Test void testValidRelNodeQuery() throws SQLException {
+    final RelNode relNode = builder
+        .scan("test", "entries")
+        .project(builder.field("name"))
+        .build();
+    runQuery(relNode);
+  }
+
+  @Test void testRelNodeQueryException() throws SQLException {
```

Comment:
There are test failures in CI (Windows). Please fix the problem with the line endings; check other tests cases comparing plans.

Reply:
Yes. I will fix this.